modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/healthcheck/declaration/PeerDeclarationProposalHandler.scala

```diff
@@ -6,14 +6,14 @@ import scala.reflect.runtime.universe.TypeTag
 
 import org.tessellation.kryo.KryoSerializer
 import org.tessellation.sdk.domain.healthcheck.consensus.HealthCheckConsensus
-import org.tessellation.sdk.infrastructure.gossip.{IgnoreSelfOrigin, RumorHandler}
+import org.tessellation.sdk.infrastructure.gossip.RumorHandler
 
 object PeerDeclarationProposalHandler {
 
   def make[F[_]: Async: KryoSerializer, K: TypeTag](
     healthCheck: HealthCheckConsensus[F, Key[K], Health, Status[K], Decision]
   ): RumorHandler[F] =
-    RumorHandler.fromPeerRumorConsumer[F, Status[K]](IgnoreSelfOrigin) { rumor =>
+    RumorHandler.fromCommonRumorConsumer[F, Status[K]] { rumor =>
```

Comment:
How do we verify if proposal is coming actually from the author of proposal? With `PeerRumor` we have a guarantee that the rumor is authored by the `rumor.origin`, and it should be passed to the `handleProposal`.

Reply:
Fixed :+1: